src/input_common/mouse/mouse_input.h

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Yuzu Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+#include <algorithm>
+#include <functional>
+#include <mutex>
+#include <thread>
+#include <unordered_map>
+#include "common/common_types.h"
+#include "common/threadsafe_queue.h"
+#include "core/frontend/input.h"
+#include "input_common/main.h"
+#include "input_common/motion_input.h"
+
+namespace MouseInput {
+
+enum class MouseButton {
+    Left,
+    Weel,
+    Right,
+    Left2,
+    right2,
+    Undefined,
+};
```

Comment:
```suggestion
enum class MouseButton {
    Left,
    Wheel,
    Right,
    Left2,
    Right2,
    Undefined,
};
```

What is Left2 and Right2? IIRC Qt only supports the following buttons:

Left, Right, Wheel, Forward, Backward

Forward and Backward are these keys here 
![image](https://user-images.githubusercontent.com/39850852/99491488-d6e2e200-2939-11eb-83a8-be456a0d0307.png)